docs/API/panel/alertGroups/index.md

```diff
@@ -13,15 +13,10 @@ grafonnet.panel.alertGroups
 ## Index
 
 * [`fn new(title)`](#fn-new)
+* [`fn withGridPos(h="null", w="null", x="null", y="null")`](#fn-withgridpos)
 * [`obj datasource`](#obj-datasource)
   * [`fn withType(value)`](#fn-datasourcewithtype)
   * [`fn withUid(value)`](#fn-datasourcewithuid)
-* [`obj gridPos`](#obj-gridpos)
-  * [`fn withH(value=9)`](#fn-gridposwithh)
-  * [`fn withStatic(value=true)`](#fn-gridposwithstatic)
```

Comment:
I did a quick grep for `gridPos.static` in grafana/grafana doesn't yield anyting except for the definition. I assume it is not used and just leftover from something historical.

I asked internally about this.

Reply:
Answer from internal: The `static` field is only used by the typescript around it, not user-accessible.